clients/export/export.go

```diff
@@ -128,27 +118,86 @@ func (c Client) Export(ctx context.Context, params *Params) error {
 	if err != nil {
 		return fmt.Errorf("failed to export template: %w", err)
 	}
-	if err := writeTemplate(params.Out, params.OutEncoding, tmpl); err != nil {
+	if err := params.OutParams.writeTemplate(tmpl); err != nil {
 		return fmt.Errorf("failed to write exported template: %w", err)
 	}
 	return nil
 }
 
-func writeTemplate(out io.Writer, encoding OutEncoding, template []api.TemplateEntry) error {
-	switch encoding {
-	case JsonEncoding:
-		enc := json.NewEncoder(out)
-		enc.SetIndent("", "\t")
-		return enc.Encode(template)
-	case YamlEncoding:
-		enc := yaml.NewEncoder(out)
-		for _, entry := range template {
-			if err := enc.Encode(entry); err != nil {
-				return err
+type AllParams struct {
+	OutParams
+
+	OrgId   string
+	OrgName string
+
+	LabelFilters []string
+	KindFilters  []ResourceType
+}
+
+func (c Client) ExportAll(ctx context.Context, params *AllParams) error {
+	if params.OrgId == "" && params.OrgName == "" && c.ActiveConfig.Org == "" {
+		return clients.ErrMustSpecifyOrg
+	}
+
+	orgId := params.OrgId
+	if orgId == "" {
```

Comment:
Does the server require an org ID specifically and not work with an org name?

Reply:
For this API, yes